HeterogeneousCore/SonicTriton/interface/TritonService.h

```diff
@@ -122,7 +123,7 @@ class TritonService {
   void preModuleConstruction(edm::ModuleDescription const&);
   void postModuleConstruction(edm::ModuleDescription const&);
   void preModuleDestruction(edm::ModuleDescription const&);
-  void preBeginJob(edm::ProcessContext const&);
+  void preBeginJob(edm::PathsAndConsumesOfModulesBase const&, edm::ProcessContext const&);
```

Comment:
@trevin-lee This change would prevent the code from compiling. Are these some un-wanted changes from a developing branch.